mysql-test/suite/innodb/t/percona_changed_page_bmp_requests_debug.test

```diff
@@ -68,6 +68,10 @@ EOF
 --let $assert_cond = @@innodb_file_per_table = 0
 --source include/assert.inc
 
+# Read current state of bitmaps after restarts
+--let $bmp_data_dir = $CUSTOM_INNODB_DATA_HOME_DIR
+--source include/percona_get_last_bmp_file_name_size.inc
+
```

Comment:
Since this is debug test, I would have skipped checkpoints to get stability SET GLOBA innodb_checkpooint_disable=TRUE;

As you have already added the .inc logic, OK to keep this.